src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -62,19 +67,22 @@ public function __construct(TranslationWriterInterface $writer, TranslationReade
     protected function configure()
     {
         $this
-            ->setDefinition(array(
-                new InputArgument('locale', InputArgument::REQUIRED, 'The locale'),
-                new InputArgument('bundle', InputArgument::OPTIONAL, 'The bundle name or directory where to load the messages, defaults to app/Resources folder'),
-                new InputOption('prefix', null, InputOption::VALUE_OPTIONAL, 'Override the default prefix', '__'),
-                new InputOption('output-format', null, InputOption::VALUE_OPTIONAL, 'Override the default output format', 'yml'),
-                new InputOption('dump-messages', null, InputOption::VALUE_NONE, 'Should the messages be dumped in the console'),
-                new InputOption('force', null, InputOption::VALUE_NONE, 'Should the update be done'),
-                new InputOption('no-backup', null, InputOption::VALUE_NONE, 'Should backup be disabled'),
-                new InputOption('clean', null, InputOption::VALUE_NONE, 'Should clean not found messages'),
-                new InputOption('domain', null, InputOption::VALUE_OPTIONAL, 'Specify the domain to update'),
-            ))
+            ->setDefinition(
+                array(
+                    new InputArgument('locale', InputArgument::REQUIRED, 'The locale'),
+                    new InputArgument('bundle', InputArgument::OPTIONAL, 'The bundle name or directory where to load the messages, defaults to app/Resources folder'),
+                    new InputOption('prefix', null, InputOption::VALUE_OPTIONAL, 'Override the default prefix', '__'),
+                    new InputOption('output-format', null, InputOption::VALUE_OPTIONAL, 'Override the default output format', 'yml'),
+                    new InputOption('dump-messages', null, InputOption::VALUE_NONE, 'Should the messages be dumped in the console'),
+                    new InputOption('force', null, InputOption::VALUE_NONE, 'Should the update be done'),
+                    new InputOption('no-backup', null, InputOption::VALUE_NONE, 'Should backup be disabled'),
+                    new InputOption('clean', null, InputOption::VALUE_NONE, 'Should clean not found messages'),
+                    new InputOption('domain', null, InputOption::VALUE_OPTIONAL, 'Specify the domain to update'),
+                )
```

Comment:
You should also revert the CS change here to avoid making merging more complex with older branches.